site/content/docs/monitoring/monitoring-local-environments.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Monitoring local environments
+weight: 7
+menu:
+  docs:
+    parent: "Monitoring"
+---
+
+Checkly allows you to check your publicly accessible websites and APIs. However, by default Checkly won't have access to sites that aren't publicly available (for instance localhost). 
+
+With [ngrok](https://ngrok.com/) you can secure a tunnel to your localhost, so you can run checks on your internal sites directly, without the need of deploying every change.
+
+Here 
+
+1. Go to [https://ngrok.com/](https://ngrok.com/) and create an account. 
+2. Download Ngrok, install it and get it started [https://dashboard.ngrok.com/get-started/setup](https://dashboard.ngrok.com/get-started/setup)
```

Comment:
Let's avoid having naked links like this when we can. Make the most relevant part of the text link to the URL, e.g.: [get it started](https://dashboard.ngrok.com/get-started/setup)